src/libwalletqt/WalletManager.cpp

```diff
@@ -1,6 +1,7 @@
 #include "WalletManager.h"
 #include "Wallet.h"
 #include "wallet/wallet2_api.h"
+#include <boost/lexical_cast.hpp>
```

Comment:
Gives build error (os x). Had to add path to boost to monero-core.pro
Not sure if that is needed on other platforms.